usaspending_api/awards/v2/views/transactions.py

```diff
@@ -68,8 +68,18 @@ def _business_logic(self, request_data: dict) -> list:
         lower_limit = (request_data["page"] - 1) * request_data["limit"]
         upper_limit = request_data["page"] * request_data["limit"]
 
-        queryset = TransactionNormalized.objects.all().values(*list(self.transaction_lookup.values())).filter(**filter)
-
+        queryset = (
+            TransactionNormalized.objects.all()
+            .values(*list(self.transaction_lookup.values()))
```

Comment:
I haven't tried it, might be cleaner to use `.select_related('transaction_fabs')` as long as it translated to a left join

Reply:
good call, it makes it much cleaner.